js/components/urlBar.js

```diff
@@ -38,16 +38,14 @@ class UrlBar extends ImmutableComponent {
     this.onFocus = this.onFocus.bind(this)
     this.onBlur = this.onBlur.bind(this)
     this.onKeyDown = this.onKeyDown.bind(this)
-    this.onCut = this.onCut.bind(this)
-    this.onKeyUp = this.onKeyUp.bind(this)
+    this.onChange = this.onChange.bind(this)
```

Comment:
this is minor, but the `s/onChange/onKeyUp` change should be kept because it helps readability